cogs/Moderation.py

```diff
@@ -203,7 +201,7 @@ async def clear_old_cached_data(self):
         # this algorithm finds the number of messages the delete, then deletes them in bulk
         n_fingerprints_to_delete = 0
         for fingerprint in self.fingerprints:
-            if time.time() - fingerprint.created_at > 60:
+            if time.time() - fingerprint.created_at > 120:
                 n_fingerprints_to_delete += 1
```

Comment:
See discussion about this change [here](https://discord.com/channels/238956364729155585/1040360228388098159/1046657929052422214)